gtfs/gtfs_utils/gtfs_utils/config.schema.json

```diff
@@ -0,0 +1,145 @@
+    "title": "Main configuration object parameter",
+    "description": "The configuration for a gtfs_stats run",
+    "type": "object",
+    "properties": {
+      "files": {
+        "$ref": "#/definitions/files"
+      },
+      "s3": {
+        "$ref": "#/definitions/s3"
+      },
+      "date_range": {
+        "description": "First and last date of the gtfs files to be download from bucket. {Format: *YYYY-MM-DD*}",
+        "type": "array",
+        "items": {
+          "type": "string",
+          "pattern": "^\\d{4}-\\d{2}-\\d{2}$"
+        },
+        "maxLength": 2,
+        "minLength": 2
+      },
+      "use_data_from_today": {
+        "description": "Download only gtfs data from today (overrides date_range). {default: *False*}",
+        "type": "boolean"
+      },
+      "max_gtfs_size_in_mb": {
+        "description": "Limit the maximum size of the downloaded gtfs files (in MB). If not set the limit is only free disk space.",
+        "type": "integer"
+      },
+      "display_download_progress_bar": {
+        "description": "If true displays a proggress bar while downloading. {default: *False*}",
+        "type": "boolean"
+      },
+      "display_size_on_progress_bar": {
+        "description": "If true displays file size on the download status bar. {default: *False*}",
+        "type": "boolean"
+      },
+      "delete_downloaded_gtfs_zip_files": {
+        "description": "If true delete the gtfs files after parsing them. {default: *True*}",
+        "type": "boolean"
+      },
+      "write_filtered_feed": {
+        "description": "If true writes a filtered version of the gtfs for the specific dates. {default: *False*}",
+        "type": "boolean"
+      },
+      "console_verbosity": {
+        "description": "Lowest logging level to be printed to console. {default: *'Error'*}",
+        "type": "string"
+      }
+    },
+    "required": [
+      "date_range",
+      "files",
+      "s3"
+    ],
+    "additionalProperties": false
+  },
+  "$comment": "The definitions are separated to a different key so it won't appear in the main doc",
+  "definitions": {
+    "files": {
+      "title": "Files object parameters",
+      "$$target": "#/definitions/files",
+      "description": "files info would be under the `files` tag.",
+      "type": "object",
+      "properties": {
+        "base_directory": {
+          "description": "base directory for the for the created files.",
+          "type": "string"
+        },
+        "output_file_name_regexp": {
+          "description": "A regular expression used to find existing output files.",
+          "type": "string"
+        },
+        "output_file_type": {
+          "description": "The file type for the outputs (for example, csv.gz).",
+          "type": "string"
+        },
+        "child_directories": {
+          "type": "object",
+          "properties": {
+            "gtfs_feeds": {
+              "description": "The name of the directory for the gtfs_feeds.",
```

Comment:
"gtfs_feeds" -> "gtfs feeds"
Also, it'd be better to note that this is the directory the GTFS files **will be downloaded to**.